packages/suite/src/views/settings/device/index.tsx

```diff
@@ -199,13 +217,32 @@ const Settings = () => {
                     <TextColumn
                         title={<Translation id="TR_FIRMWARE_VERSION" />}
                         description={
-                            <>
+                            <Version>
                                 <Translation
                                     id="TR_YOUR_CURRENT_FIRMWARE"
-                                    values={{ version: getFwVersion(device) }}
+                                    values={{
+                                        version: (
+                                            <VersionTooltip content={getFwRevision(device) || ''}>
+                                                <VersionLink
+                                                    target="_blank"
+                                                    href={`https://github.com/trezor/trezor-firmware/commit/${getFwRevision(
+                                                        device,
+                                                    )}`}
+                                                >
+                                                    <VersionButton
+                                                        variant="tertiary"
+                                                        icon="EXTERNAL_LINK"
+                                                        alignIcon="right"
+                                                    >
+                                                        {getFwVersion(device)}
```

Comment:
This function is called 3 times here, would be best to assign the result to a variable an use it here.

Reply:
`getFwVersion` is only used once in this file. `getFwRevision` is used twice so I am moving that to a variable.

Reply:
Indeed, sorry! I should start wearing my glasses more often :smile:

Reply:
14d766f